installer/scripts/omsadmin.sh

```diff
@@ -353,6 +353,15 @@ onboard_scom()
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
     configure_logrotate
+
+    #Open port 1270 if not already open
+    IS_PORT_OPEN=`cat $OMI_CONF_FILE | grep httpsport | grep 1270 | wc -l`
```

Comment:
Can this search be more exact? According to the [Microsoft docs](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-linux-agents), the exact line should be `httpsport=1270`